ci/init-plugin-test-service.sh

```diff
@@ -22,8 +22,8 @@ after() {
     docker exec -i apache-apisix_kafka-server2_1 /opt/bitnami/kafka/bin/kafka-topics.sh --create --zookeeper zookeeper-server2:2181 --replication-factor 1 --partitions 1 --topic test4
 
     # prepare openwhisk env
-    docker pull openwhisk/action-nodejs-v14:nightly
-    docker run --rm -d --name openwhisk -p 3233:3233 -p 3232:3232 -v /var/run/docker.sock:/var/run/docker.sock openwhisk/standalone:nightly
+    docker pull openwhisk/action-nodejs-v14:7545841
```

Comment:
What version does this id represent?

Reply:
@monkeyDluffy6017, it does not represent any version I suppose because, in the openwhisk docker hub, the tags were specified like random numbers (no semantically versioned tags were available). I just found a combination of tags that worked well.

Reply:
It's very werid

Reply:
can we use `docker pull openwhisk/action-nodejs-v14:latest`?